frontend/src/layout/navigation-3000/navigationLogic.tsx

```diff
@@ -312,8 +314,12 @@ export const navigation3000Logic = kea<navigation3000LogicType>([
             (isNavCollapsedDesktop, mobileLayout): boolean => !mobileLayout && isNavCollapsedDesktop,
         ],
         navbarItems: [
-            () => [featureFlagLogic.selectors.featureFlags],
-            (featureFlags): NavbarItem[][] => {
+            () => [
+                featureFlagLogic.selectors.featureFlags,
+                dashboardsModel.selectors.dashboardsLoading,
+                dashboardsModel.selectors.pinnedDashboards,
```

Comment:
Out of scope for this PR but still a shame that we are loading all dashboards on every page load, regardless of whether they are shown or not....

Reply:
Yup, there's a couple of places we do this. Would also be great to fetch the pinned dashboards from the API instead of filtering ALL dashboards locally 😅